.gitattributes

```diff
@@ -0,0 +1,4 @@
+.git* export-ignore
+.circleci export-ignore
+.drone* export-ignore
+saxon-license.lic export-ignore
```

Comment:
Are these really necessary? I thought git archive would respect ignored files, so you would only need to include files here that are part of the git repository. Anything in .gitignore is automatically excluded. Am I misunderstanding how git archive works?

Reply:
@adborden You are correct. The git archive already ignores the files in .gitignore. When git action does a checkout these files don't exist in the repo so no need to exclude them again.

I removed the ignore for  saxon-license.lic.